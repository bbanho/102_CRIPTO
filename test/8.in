-5
13524
IHQ*NS!*ZTT*PE$*%25*369*7:8*;>2*$C\*J"F*ZH3