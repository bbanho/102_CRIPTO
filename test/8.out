HELLO WORLD! 01234567890 BYE BYE.